docs/en/observability/monitor-k8s/monitor-k8s-application-performance.asciidoc

```diff
@@ -41,6 +41,7 @@ like when directing traces from multiple applications to separate {es} clusters.
 === Step 2: Save your secret token
 
 A {observability-guide}/secret-token.html[secret token] is used to secure communication between APM agents
+
 and APM Server. On the {ecloud} deployment page, select *APM* and copy your APM Server secret token.
 To avoid exposing the secret token, you can store it in a Kubernetes secret. For example:
```

Comment:
This is out of date. Here are instructions for finding the APM Secret token: https://www.elastic.co/guide/en/observability/current/secret-token.html#create-secret-token